firestore/integration_test_internal/src/query_test.cc

```diff
@@ -660,6 +660,7 @@ TEST_F(QueryTest, TestQueriesCanUseArrayContainsFilters) {
 }
 
 TEST_F(QueryTest, TestQueriesCanUseInFilters) {
+  GTEST_SKIP() << "Skipping this test until double free bug is fixed.";
```

Comment:
Could you add the b/# here so we can track it?

Reply:
+1. Also, please shorten "Skipping this test" to just "Skip".

Reply:
done